Octokit/Models/Response/PullRequest.cs

```diff
@@ -48,6 +49,11 @@ public PullRequest(Uri url, Uri htmlUrl, Uri diffUrl, Uri patchUrl, Uri issueUrl
         }
 
         /// <summary>
+        /// The universal Id (different from the pull request number) this pull request.
+        /// </summary>
+        public int Id { get; protected set; }
```

Comment:
And in terms of the TravisCI build sometimes we get an unrelated problem where it blows up in TravisCI and is not related to your changes.  We have to kick off another build (or the next commit pushed will obviously trigger a new build as well).  I see your latest commit, the builds have all passed :+1: